pulser/sequence.py

```diff
@@ -825,8 +840,38 @@ def target(
             channel (str): The channel's name provided when declared. Must be
                 a channel with 'Local' addressing.
         """
+        qubits = cast(QubitId, qubits)
+
         self._target(qubits, channel)
 
+    @_verify_parametrization
+    def target_index(
+        self,
+        qubits: Union[int, Iterable[int], Parametrized],
+        channel: Union[str, Parametrized],
+    ) -> None:
+        """Acts as ``target`` using indices instead of Qubit IDs.
```

Comment:
Yes, I would prefer it, I think it's better that things are kept self-explanatory. In an IDE, I would prefer that the docstring gave me all the information, instead of me having to go look at the docstring of another function.